src/UI/Components/Diagram/actions.ts

```diff
@@ -13,9 +14,14 @@ import { Colors, EntityConnection, ServiceEntityBlock } from "./shapes";
  * https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#linkTools
  *
  * @param {dia.LinkView} linkView  - The view for the joint.dia.Link model.
+ * @param {(): void} linkView  - The view for the joint.dia.Link model.
```

Comment:
I think you forgot to adjust the copy pasted line ;) 
you got two arguments missing in the docs